### PR TITLE
[13.0][MIG] account_invoice_import

### PR DESCRIPTION
**Pull request by yvaucher**
Replaces https://github.com/OCA/edi/pull/213

### CODE REVIEW

**Comment by bosd** [comment]
@yvaucher You did some improvements on the module in this pr. Are there any commits which can be forward ported to V14?

File: account_invoice_import/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "name": "Account Invoice Import",
+    "version": "13.0.1.0.0",
+    "category": "Accounting & Finance",
+    "license": "AGPL-3",
+    "summary": "Import supplier invoices/refunds as PDF or XML files",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": [
+        "account",
+        "base_iban",
```
Comment (sonhd91) [inline account_invoice_import/__manifest__.py:14]:
```suggestion
        "edi",
        "base_iban",
```

**Comment by bosd** [comment]
@yvaucher Can you force push

**Pull request closed without merging**